catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/criteria/contextual/ContextualEvaluator.java

```diff
@@ -337,15 +336,15 @@ private static String getIndexableText(String document, String[] xpathSelectors)
                 }
             }
         } catch (XPathExpressionException e1) {
-            LOGGER.catching(e1);
+            LOGGER.debug("XPathExpressionException");
```

Comment:
It would also be more beneficial to state why the exception is thrown (i.e. unable to evaluate xpath)  `XPathExpressionException` will already be logged when you log `e1`.